sdk/core/Azure.Core.TestFramework/src/ClientDiagnosticListener.cs

```diff
@@ -146,7 +146,8 @@ public void Dispose()
                 // traverse the activities and check for duplicates among ancestors
                 while (activity != null)
                 {
-                    if (operationName == activity.Parent?.OperationName)
+                    // TODO find a better way to avoid the duplicate Azure.Core spans when using Test-Proxy
```

Comment:
Why do you even have the problem with duplicated spans?

Reply:
Must have been a bug that has since been fixed - removed.